### ISSUE TITLE
Clear the history

### ISSUE THREAD

**Issue by massinlaaouaj** [timvisee/ffsend#74]
Hi again Tim, how I can clear all my history?

I tried: ffsend delete --history, but I can't cause it need a URL.

**Comment by timvisee** [comment]
There currently is no command available to delete history.

For now, you can manually delete the file the history is stored in, or remove file entries from it. On Linux/macOS machines, this file is located at `~/.cache/ffsend/history.toml`. You could use this command to delete that file:  
```bash
rm ~/.cache/ffsend/history.toml
```
Note that for the `snap` installation option, this file is stored at a different location, inside the file tree specific to the `snap` package. I'm not sure how to properly delete the file in that case.

Deleting the file using `ffsend delete <URL>` also removes the link from history at this moment. You might be able to use this as temporary workaround.

I've opened the following issue a while back to implement a command to clear history. I might be able to work on this later today:  
https://gitlab.com/timvisee/ffsend/issues/2

**Comment by massinlaaouaj** [comment]
Thanks Tim I used and it works, by the way I found for the `snap` is at this location:
`
rm ~/snap/ffsend/26/.cache/ffsend/history.toml
`

**Comment by timvisee** [comment]
In the meanwhile I've added a simple flag/option to the history sub-command for history management:

```bash
# Clear all history
ffsend history --clear

# Remove history item
ffsend history --rm https://send.firefox.com/#sample-url
```

This is released as part of `v0.2.49`, which should be available as soon as this release pipeline succeeds:  
https://gitlab.com/timvisee/ffsend/pipelines/68631447

**Comment by massinlaaouaj** [comment]
Oh great Tim that's nice!!

Thank you very much for your time.
